Wikipedia/UI-V5/WMFArticleListCollectionViewController.m

```diff
@@ -130,6 +143,21 @@ - (void)scrollToArticleIfOffscreen:(MWKArticle*)article animated:(BOOL)animated
 
 #pragma mark - UIViewController
 
+- (void)loadView {
```

Comment:
Should be needed if you just use the initializer for passing in the collection view layout desired. 

You should only need this to use a custom UICollectionView subclass. `loadView` isn't required if all you need to do is configure the collection view.